open_lm/main.py

```diff
@@ -26,10 +26,8 @@
     FullStateDictConfig,
     StateDictType,
     CPUOffload,
-    ShardingStrategy,
```

Comment:
(we're importing this twice, removing it in this PR because I saw it and was an easy fix)